doc/nrf/samples/bl.rst

```diff
@@ -9,3 +9,4 @@ Bluetooth samples
    :glob:
 
    ../../../samples/bluetooth/*/README
+   ../../../samples/bluetooth/fast_pair/*/README
```

Comment:
is it possible to sort the displayed sample list alphabetically? It seems that the Fast Pair sample is currently at the very bottom of the list

Reply:
yeah, I can check this ,and can be improved later.